.cirrus.yml

```diff
@@ -103,6 +103,16 @@ build_task:
               # ID for re-use of build output
               CI_DESIRED_RUNTIME: crun
               CI_DESIRED_NETWORK: netavark
+              CI_DESIRED_DATABASE: boltdb
+        - env: &sqliteenvvars
+              DISTRO_NV: ${FEDORA_NAME}
+              # Not used here, is used in other tasks
+              VM_IMAGE_NAME: ${FEDORA_CACHE_IMAGE_NAME}
+              CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+              # ID for re-use of build output
+              CI_DESIRED_RUNTIME: crun
+              CI_DESIRED_NETWORK: netavark
+              CI_DESIRED_DATABASE: sqlite
```

Comment:
This will very likely fail at the moment (they didn't pass locally last week).  Can we keep "boltdb" here for the time being and flip it over to "sqlite" with a PR that lets the integration tests pass with it?